mapr_devsandbox_container_setup.sh

```diff
@@ -1,9 +1,22 @@
 #!/bin/bash
-
 set -x
+IMAGE="maprtech/dev-sandbox-container:latest"
+INTERFACE="en0"
+while [ $# -gt 0 ]
+do
+  case "$1" in
+  -image) shift; IMAGE=$1;;
+  -nwiterface) shift; INTERFACE=$1;;
+  esac
+  shift
+done
 
-IMAGE="${1:-"maprtech/dev-sandbox-container:latest"}"
-IP=$(ipconfig getifaddr en0)
+which ipconfig
```

Comment:
The which can be inlined into the if to make this a clearer idiom and allow the use of set -euxo pipefail